nullaway/src/main/java/com/uber/nullaway/handlers/LibraryModelsHandler.java

```diff
@@ -156,125 +154,174 @@ private static LibraryModels loadLibraryModels() {
+                0)
+            .put(methodRef("java.io.File", "File(java.lang.String)"), 0)
+            .put(methodRef("java.lang.Class", "getResource(java.lang.String)"), 0)
+            .put(methodRef("java.lang.Class", "isAssignableFrom(java.lang.Class<?>)"), 0)
+            .put(methodRef("java.lang.System", "getProperty(java.lang.String)"), 0)
+            .put(
+                methodRef(
+                    "java.net.URLClassLoader",
+                    "newInstance(java.net.URL[],java.lang" + ".ClassLoader)"),
+                0)
+            .put(
+                methodRef(
+                    "javax.lang.model.element.Element", "<A>getAnnotation(java.lang.Class<A>)"),
+                0)
+            .put(
+                methodRef(
+                    "javax.lang.model.util.Elements",
+                    "getPackageElement(java.lang" + ".CharSequence)"),
+                0)
+            .put(
+                methodRef(
+                    "javax.lang.model.util.Elements",
+                    "getTypeElement(java.lang" + ".CharSequence)"),
+                0)
+            .put(
+                methodRef(
+                    "javax.lang.model.util.Elements",
+                    "getDocComment(javax.lang.model" + ".element.Element)"),
+                0)
+            .put(methodRef("java.util.Deque", "addFirst(E)"), 0)
+            .put(methodRef("java.util.Deque", "addLast(E)"), 0)
+            .put(methodRef("java.util.Deque", "offerFirst(E)"), 0)
+            .put(methodRef("java.util.Deque", "offerLast(E)"), 0)
+            .put(methodRef("java.util.Deque", "add(E)"), 0)
+            .put(methodRef("java.util.Deque", "offer(E)"), 0)
+            .put(methodRef("java.util.Deque", "push(E)"), 0)
+            .put(methodRef("java.util.Collection", "<T>toArray(T[])"), 0)
+            .put(methodRef("java.util.ArrayDeque", "addFirst(E)"), 0)
+            .put(methodRef("java.util.ArrayDeque", "addLast(E)"), 0)
+            .put(methodRef("java.util.ArrayDeque", "offerFirst(E)"), 0)
+            .put(methodRef("java.util.ArrayDeque", "offerLast(E)"), 0)
+            .put(methodRef("java.util.ArrayDeque", "add(E)"), 0)
+            .put(methodRef("java.util.ArrayDeque", "offer(E)"), 0)
+            .put(methodRef("java.util.ArrayDeque", "push(E)"), 0)
+            .put(methodRef("java.util.ArrayDeque", "<T>toArray(T[])"), 0)
             .build();
 
-    private static final ImmutableSetMultimap<MemberName, Integer> NULL_IMPLIES_TRUE_PARAMETERS =
-        new ImmutableSetMultimap.Builder<MemberName, Integer>()
-            .put(member(Strings.class, "isNullOrEmpty"), 0)
-            .put(member("android.text.TextUtils", "isEmpty"), 0)
+    private static final ImmutableSetMultimap<MethodRef, Integer> NULL_IMPLIES_TRUE_PARAMETERS =
+        new ImmutableSetMultimap.Builder<MethodRef, Integer>()
+            .put(MethodRef.methodRef(Strings.class, "isNullOrEmpty(java.lang.String)"), 0)
```

Comment:
Why is this one alone not just `methodRef`?

Reply:
some IntelliJ weirdness; will fix both 😄